### PR TITLE
clicking on logo now will take you to home page

### PR DESCRIPTION
**Pull request by batistaDev1113**
Changes made:

1. Image has been replaced and renamed to logo1.png

2. added cursor: pointer in App.css file 
3. now the logo will take the user to the home page when clicked.

Please let me know if you have any questions!

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/mycovidconnect/91)

### CODE REVIEW

File: .vscode/settings.json
Comment (codecakes) [inline .vscode/settings.json:1]: pls add .vscode to .gitignore ; we don't want to check-in any personal ide preferences and settings.

File: src/App.css
Comment (codecakes) [inline src/App.css:37]: you can name it `main-nav-logo`

**Review by codecakes — CHANGES_REQUESTED** [review]
@batistaDev1113 pls take a fresh merge from master to your branch and recommit with requested changes.

**Review by codecakes — CHANGES_REQUESTED** [review]
@batistaDev1113 functionally it works. you can see it here: https://5f75f42e191e98000816aff9--dreamy-keller-b8ad36.netlify.app/ ; just make the changes requested. thanks

File: .vscode/settings.json
Comment (batistaDev1113) [reply .vscode/settings.json:1]: Hi @codecakes ,

my bad....totally oversaw that.  Will fix now!

File: src/App.css
Comment (batistaDev1113) [reply src/App.css:37]: Consider it done!

**Comment by batistaDev1113** [comment]
@codecakes Everything has been resolved as requested.  For some reason, I the stylesheet lint test is failing.  I just replaced what was asked of me and the lint is now complaining even about the white color being expected to be #fff and it shows as #ffffff; which is a weird rule.
Could you please take a look at this and let me know?
Thank you in advance!

**Comment by batistaDev1113** [comment]
@codecakes I just came across an already opened issue about this from two days ago:
https://github.com/Xcov19/mycovidconnect/issues/84

I guess I can wait until that PR is done and merged and then do git pull before resubmitting :-)

Let me know if that works

File: .vscode/settings.json
Comment (codecakes) [reply .vscode/settings.json:1]: this file is still here. try git rm --cached <this_file>

**Review by codecakes — CHANGES_REQUESTED** [review]
make the change and check if the vscode file has been removed from the src check-in then make the commit. no worries on stylelint, there're another PR that fixes that.

**Comment by batistaDev1113** [comment]
Hey @codecakes ,

The folder .vscode has already been removed and changes pushed!  Please let me know if there is any issue still!

Thanks

**Review by codecakes — APPROVED** [review]
pls make a fresh merge from main repo master to ur branch before merging code;

**Pull request merged**
